src/components/SignUp.js

```diff
@@ -34,7 +34,8 @@ export class SignUp extends React.Component {
         this.setState({ error: "Invalid username and/or password" });
       }
     } catch (err) {
-      this.setState({ error: err.response.data });
+      this.setState({ error: "Invalid username and/or password" });
```

Comment:
why did you make this change?  We want to send google's response back to the user to tell them what they need to change.